common-dev.css

```diff
@@ -0,0 +1,201 @@
+header {
```

Comment:
confused here. changes to both common-dev.css and common.css
Which version does your code commit use?